spec/spec_helper.rb

```diff
@@ -48,3 +48,7 @@ def when_current_user_is(user)
 def format_time(time)
   time.strftime('%Y-%m-%d %H:%M %P')
 end
+
+def format_datetime(datetime)
+  datetime.strftime('%A, %B %e, %Y - %l:%M %P').squish
```

Comment:
What's the `squish` for? I know what it does, but why do you need it?

Reply:
I copied the method over.

Reply:
Seems like you could remove it then?

Reply:
Or maybe not — but we should make sure we know why it's needed (or not)

Reply:
ok